packages/flutter_tools/lib/src/base/error_handling.dart

```diff
@@ -2,17 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
-import 'dart:io' as io show Directory, File, Link;
+import 'dart:convert'; // ignore: dart_convert_import
```

Comment:
What's the relationship between these `ignore`s and `https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/forbidden_imports_test.dart` ?

Reply:
Oh I see, this file was added as an exception but that changed when I renamed it